docs/src/literate-tutorials/transient_heat_equation.jl

```diff
@@ -209,8 +209,9 @@ for t in Δt:Δt:T
     #Finally, we can solve the time step and save the solution afterwards.
     u = A \ b
 
-    addstep!(pvd, t) do io
-        write_solution(io, dh, u)
+    VTKGridFile("transient-heat-$step", dh) do vtk
```

Comment:
PR looks good but just skimming through this, perhaps this should be `open_vtk` instead?

Reply:
What do you mean? That we have some function `open_vtk` which supportes the do syntax and which automatically figures out the correct format?

Reply:
I just mean it might look better to have something like
```julia
open_vtk("transient-heat-$step", dh) do vtk::VTKGridFile
    write_solution(vtk, dh, u)
    pvd[t] = vtk
end
```
instead of the current
```julia
VTKGridFile("transient-heat-$step", dh) do vtk::VTKGridFile
    write_solution(vtk, dh, u)
    pvd[t] = vtk
end
```

Reply:
But how does `open_vtk` figure out that it is a `VTKGridFile` (which maps for us to VTU) and not e.g. a multiblock file or structured mesh file?

Reply:
It doesn't figure it out, it always return it. Perhaps `open_vtk(VTKBlockFile, ....)` could be the interface or something.

Reply:
> PR looks good but just skimming through this, perhaps this should be open_vtk instead?

While I agree that it looks better, it also looks like it comes from WriteVTK? So in that case, perhaps don't export but make public? Having to write `Ferrite.open_vtk` is perhaps equally good as `VTKGridFile`, but looks nicer (and is even more clear that comes from Ferrite)?

Sharing some more thoughts below, to make those notes easier to find in the future.